Sources/App/Settings/AppleWatch/ComplicationEditViewController.swift

```diff
@@ -184,6 +189,11 @@ class ComplicationEditViewController: FormViewController, TypedRowControllerType
             cell.detailTextLabel?.text = row.value?.style
         }
 
+        <<< SwitchRow("IsPrivate") {
+            $0.title = L10n.Watch.Configurator.Rows.IsPrivate.title
+            $0.value = self.config.IsPrivate
```

Comment:
I think it is worth flipping the way this is presented. "Show When Locked <default on>" is a little easier to mentally discern than the double-negative "Hide when locked <default off>".